ssh/proxy.go

```diff
@@ -216,7 +215,7 @@ func (p *ProxyConn) sendOKMsg(key PublicKey) error {
 	return p.Downstream.transport.writePacket(Marshal(&okMsg))
 }
 
-func (p *ProxyConn) sendFailureMsg(method string) error {
+func (p *ProxyConn) SendFailureMsg(method string) error {
```

Comment:
Export send failure message function for sshr.
Now, sshr can use this function when fetch upstream failed. So client can see a reason what makes connection closed.
And this is correct failure message type the normal ssh connection also.

ex)

- as-is (client does not know what makes failed connection)

```
$ ssh hogemoge@sample.local -p 2222
Warning: Permanently added '[xxx.xxx.xxx.xxx]:2222' (RSA) to the list of known hosts.
Connection closed by xxx.xxx.xxx.xxx port 2222
```

- to-be (client got correct failed message like permission denied by user id not exist)

```
$ ssh hogemoge@sample.local -p 2222
Warning: Permanently added '[xxx.xxx.xxx.xxx]:2222' (RSA) to the list of known hosts.
hogemoge@xxx.xxx.xxx.xxx: Permission denied (user hogemoge does not exists).
```